scheds/rust/scx_layered/src/main.rs

```diff
@@ -2147,6 +2168,239 @@ impl<'a> Scheduler<'a> {
         Ok(())
     }
 
+    fn calc_target_llcs(&self) -> Vec<usize> {
+        let nr_llcs = self.cpu_pool.topo.all_llcs.len();
+        let mut wants = vec![0; self.layers.len()];
+
+        for (i, layer) in self.layers.iter().enumerate() {
+            let owned = self.sched_stats.layer_utils[i][LAYER_USAGE_OWNED];
+            let open = self.sched_stats.layer_utils[i][LAYER_USAGE_OPEN];
+            let total_util = owned + open;
```

Comment:
When determining target number of CPUs, open consumptions are considered iff the layer has no CPUs because otherwise grouped layers end up overallocating. Also, I wonder whether it'd make more sense to determine the number of LLCs to allocate in terms of the the result of target CPU calculations.

Reply:
Note that the allocation needs to be fair within each LLC too for intel and other CPUs with one or few LLCs.